### ISSUE TITLE
I got some problems when train network

### ISSUE THREAD

**Issue by Wangs121** [experiencor/self-driving-toy-car#2]
Hello, When I run _'Learn to Drive.ipynb'_  , In **Train non-recurrent network** part of : 
`tb_counter  = max([int(num) for num in os.listdir('../logs/')]) + 1
tensorboard = TensorBoard(log_dir='../logs/' + str(tb_counter), histogram_freq=0, write_graph=True, write_images=False)

minimizer = SGD(lr=0.0001)

#model.compile(loss=custom_loss, optimizer=minimizer)
model.compile(loss=custom_loss, optimizer='adam')

model.fit_generator(generator = gen_train.get_gen(),
                    steps_per_epoch = num_train, 
                    epochs  = 3, 
                    verbose = 1,
                    validation_data = gen_valid.get_gen(), 
                    validation_steps = num_valid, 
                    callbacks = [early_stop, checkpoint, tensorboard], 
                    max_q_size = 3)`
it occurred an **withFileNotFoundError**, can't find ../logs/ folder, but when I create a folder named logs, another error occurred. Your generous help means a lot to me. Thank you.
